src/aap_eda/services/project/imports.py

```diff
@@ -163,13 +163,17 @@ def _sync_rulebook(
         rulebook_info: RulebookInfo,
     ):
         if rulebook.rulesets == rulebook_info.raw_content:
+            models.Activation.objects.filter(rulebook=rulebook).update(
```

Comment:
Minor, optimization: you can save DB queries if you store the git_hash in a variable for the later updates.